include/sampleflow/consumers/spurious_autocovariance.h

```diff
@@ -290,7 +290,8 @@ namespace SampleFlow
               current_autocovariation[i] = alpha[i];
               //current_mean.size() is equal to sample.size(). While we don't get samples here, alpha.size helps to
               for (int j=0; j<current_mean.size(); ++j) current_autocovariation[i] -= current_mean[j]* beta(i,j);
-              current_autocovariation[i] += static_cast<scalar_type>((n_samples-i))*(current_mean*current_mean).sum()/(n_samples);
+              current_autocovariation[i] += (current_mean*current_mean).sum();
+              //current_autocovariation[i] += static_cast<scalar_type>((n_samples-i-1))*(current_mean*current_mean).sum()/(n_samples); Leave it, it might be needed later.
```

Comment:
In general: Just remove code you don't need any more.

I'll just remove this line before merging your code. (github lets me edit your files, and just creates a second commit.)